silverstripe/framework/SS-2015-006-1.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SS-2015-006: XSS In GridField print
+link:      http://www.silverstripe.org/software/download/security-releases/ss-2015-006/
+cve:       ~
+branches:
+    3.1.x:
+        time:     2015-02-12 15:55:00
```

Comment:
@stof Oh indeed, I misinterpreted the contribution docs in the past. I should re-check my PRs that have been merged in the past. :(